pkg/build/strategies/sti/sti.go

```diff
@@ -115,7 +115,7 @@ func (b *STI) Build(request *api.Request) (*api.Result, error) {
 	if request.Incremental {
 		glog.V(1).Infof("Existing image for tag %s detected for incremental build.", request.Tag)
 	} else {
-		glog.V(1).Infof("Clean build will be performed")
+		glog.V(1).Infof("Clean build will be performed.")
```

Comment:
is the dot needed? I can see that we are not putting dot into other messages, we should try to make the output consistent (even if this is V(1))